justfile

```diff
@@ -27,7 +27,8 @@ APP_REL_DOCKERCOMPOSE :="docker-compose.release.yml"
 APP_REL_CONTAINER := APP_NAME + "_release"
 WEB_CONTAINER := APP_NAME +"_web"
 APP_VSN := `grep -m 1 'version:' mix.exs | cut -d '"' -f2`
-APP_BUILD := `git rev-parse --short HEAD`
+# Nix build doesn't include the .git repo
+#APP_BUILD := `git rev-parse --short HEAD`
```

Comment:
This is a problem. I'll need to set some sort fallback here when git isn't available (it doesn't appear to be in the nix build process)